matrimony_admin/views.py

```
@@ -21,10 +21,11 @@
 from U_auth.models import *
 from matrimony_admin.models import Subscription
 from U_auth.permissions import *
-from django.db.models import Count, Sum, Q
+from django.db.models import Count, Sum, F, Case, When, DecimalField
```

Ruff

Redefinition of unused `Count` from line 15

Remove definition: `Count`

(F811)